opta/commands/init.py

```diff
@@ -1,130 +1,16 @@
-from os import path
-from typing import Dict
+import webbrowser
 
 import click
-import yaml
-from click_didyoumean import DYMGroup
 
-from opta.commands.init_templates.environment.aws.template import awsTemplate
-from opta.commands.init_templates.environment.azure.template import azureTemplate
-from opta.commands.init_templates.environment.gcp.template import gcpTemplate
-from opta.commands.init_templates.service.k8s.template import k8sServiceTemplate
-from opta.commands.init_templates.template import Template
-from opta.exceptions import UserErrors
 
-EXAMPLES_DIR = path.join(path.dirname(__file__), "init_templates")
-
-
-def _write_result(file_path: str, result: dict) -> None:
-    print("--------------------------------------------\n")
-    print(f"\n\nAbout to write to {path.relpath(file_path)}:\n")
-
-    yaml_str = yaml.dump(result, sort_keys=False)
-    print(yaml_str)
-
-    ok = input("\n\nIs this OK? (yes) ") or "yes"
-    if ok == "no":
-        print("Aborted")
-        return
-    try:
-        with open(file_path, "x") as f:
-            f.write(yaml_str)
-    except FileExistsError:
-        raise UserErrors(
-            f"Output file {file_path} already exists, please select another output path"
-        )
-
-
-@click.group(cls=DYMGroup)
+@click.command()
 def init() -> None:
     """
-    This command initializes an opta configuration file. You can create an environment
-    configuration file by using the command `opta init env <cloud-provider>` and a service
-    file by using the command `opta init service <environment_file> <template_name>`.
-
-    In order to see the full list of available templates, you can
-    run `opta init env --help` or `opta init service --help` respectively.
-    """
-    pass
-
-
-ENVIRONMENT_TEMPLATES: Dict[str, Template] = {
-    "aws": awsTemplate,
-    "gcp": gcpTemplate,
-    "azure": azureTemplate,
-}
-
+    This command brings to you an interactive way of creating an opta configuration file.
```

Comment:
```suggestion
    Redirecting to `https://app.runx.dev/yaml-generator`
```

Reply:
`opta -h` would have this as the result.

`Usage: opta [OPTIONS] COMMAND [ARGS]...

  Welcome to opta, runx's cli!

Options:
  -h, --help  Show this message and exit.

Commands:
  apply              Initialize your environment or service to match the...
  configure-kubectl  Configure the kubectl CLI tool for the given cluster
  deploy             Push your new image to the cloud and deploy it in...
  destroy            Destroy all opta resources from the current config
  force-unlock       Force Unlocks a stuck lock on the current workspace
  init               Redirecting to `https://app.runx.dev/yaml-generator`
  logs               Get stream of logs from your service
  secret             Commands for manipulating secrets for a k8s service
  shell              Get a bash shell into one of the pods in your service`

So, need to have a better input message.
Plus, running `opta init` will not show any messages.